src/css/footer.css

```diff
@@ -0,0 +1,299 @@
+    flex-direction: row;
+    justify-content: space-between;
+  }
+}
+
+.footer-top-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-top-wrapper {
+    gap: 32px;
+  }
+}
+
+@media screen and (min-width: 1440px) {
+  .footer-bottom-wrapper {
+    display: flex;
+    flex-direction: row;
+    gap: 115px;
+  }
+}
+
+.footer-logo {
+  fill: var(--light);
+}
+
+.footer-socials-list {
+  display: flex;
+  gap: 12px;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-socials-list {
+  }
+}
+
+.socials-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  padding: 8px;
+  outline: 1px solid rgba(244, 244, 244, 0.1);
+  border-radius: 10px;
+}
+
+@media screen and (min-width: 768px) {
+  .socials-link {
+    width: 44px;
+    height: 44px;
+  }
+}
+
+.socials-link {
+  transition: all var(--transition);
+}
+
+.socials-link:hover,
+.socials-link:focus {
+  background-color: var(--light);
+}
+
+.socials-link:hover .socials-icon-footer,
+.socials-link:focus .socials-icon-footer {
+  stroke: var(--dark);
+  fill: var(--dark);
+}
+
+.socials-icon-footer {
+  fill: var(--light);
+}
+
+.footer-title {
+  margin-top: 40px;
+  color: var(--light);
+  font-size: 38px;
+  font-weight: 400;
+  line-height: 1.15;
+  letter-spacing: -0.76px;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-title {
+    margin-top: 0px;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 1.07143;
+    letter-spacing: -0.02em;
+    width: 400px;
+  }
+}
+
+@media screen and (min-width: 1440px) {
+  .footer-title {
+    width: 595px;
+  }
+}
+
+.accent-text {
+  font-style: italic;
+}
+
+/* #region Form */
+.subscribe-form {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+
+  gap: 20px;
+
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 1.28px;
+}
+
+@media screen and (min-width: 375px) {
+  .subscribe-form {
+    width: 335px;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .subscribe-form {
+    margin-top: 40px;
+    width: 352px;
+    font-size: 16px;
+    line-height: 1.5px;
+  }
+}
+
+@media screen and (min-width: 1440px) {
+  .subscribe-form {
+    margin-top: 0px;
+  }
+}
+
+.subscribe-form-title {
+  margin-top: 20px;
+  color: var(--light);
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 1.28;
+}
+
+@media screen and (min-width: 768px) {
+  .subscribe-form-title {
+    margin-top: 0px;
+    font-size: 18px;
+    line-height: 1.33;
+  }
+}
+
+@media screen and (min-width: 1440px) {
+  .subscribe-form-title {
+    margin-top: 0px;
+  }
+}
+
+.subscribe-form-input {
+  width: 100%;
+  padding: 12px 14px;
+  border-radius: 30px;
+  border: 1px solid var(--light);
+  background-color: var(--dark);
+
+  color: rgba(244, 244, 244, 0.6);
+  transition: all var(--transition);
+  color: var(--light);
+}
+
+.subscribe-form-input:focus,
+.subscribe-form-input:hover {
+  outline: none;
```

Comment:
Use `outline: 1px solid var(--light)`. Remove 'border-color: orange'. We already have an input on the white background and it has similar styling but the outline is --dark. The would be if just extend the existing input component with dark version and reuse most of the stylings. Styles are in the search-input.css file.

Reply:
done